<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IndirectDataManipulation</class>
 <widget class="QMainWindow" name="IndirectDataManipulation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>758</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Indirect Data Manipulation</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="twIDMTabs">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabSymmetrise">
       <attribute name="title">
        <string>Symmetrise</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabSqw">
       <attribute name="title">
        <string>S(Q, w)</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabMoments">
       <attribute name="title">
        <string>Moments</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabElwin">
       <attribute name="title">
        <string>Elwin</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabIqt">
       <attribute name="title">
        <string>I(Q, t)</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="layout_bottom">
      <item>
       <widget class="QPushButton" name="pbHelp">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Open interface help page in your web browser.</string>
        </property>
        <property name="text">
         <string>?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbPythonExport">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>25</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Export a Python script for the algorithms run on this tab.</string>
        </property>
        <property name="text">
         <string>Py</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_14">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbManageDirs">
        <property name="text">
         <string>Manage Directories</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
